WordPress/Classes/Users/Components/UserListItem.swift

```diff
@@ -18,7 +18,7 @@ struct UserListItem: View {
         } label: {
             HStack(alignment: .top) {
                 if !dynamicTypeSize.isAccessibilitySize {
-                    UserProfileImage(size: height, url: user.profilePhotoUrl)
+                    AvatarView(style: .single(user.profilePhotoUrl), placeholderImage: Image("gravatar").resizable())
```

Comment:
Can we retain the `person.circle.fill` default?

Repeating the Gravatar icon so much is a lot visually

Alternatively, IINM we have an implementation somewhere that will make a contacts-like version that's just the person's initials

Reply:
> Repeating the Gravatar icon so much is a lot visually

Do you mean the Gravatar icons in the screenshot in the PR description. Those are actually the final images, rather than default icon bundled with the app.

I believe that's a feature of `AvatarView`, where it may use an "processed URL" to fetch images instead of the original one passed into `AvatarView`. That's also the reason why you see a higher resolution avatar for the demo user in the screenshots in the PR description.

https://github.com/wordpress-mobile/WordPress-iOS/blob/eb4881828b4d21e5fbce0f0105f4ed2fd57c2967/WordPress/Classes/ViewRelated/Views/List/NotificationsList/AvatarView.swift#L82-L83

Reply:
I'll merge this PR for now. But happy to make code changes to address further comments if needed.